src/content/docs/troubleshooting.mdx

```diff
@@ -191,7 +188,7 @@ native code of your app, and are not patchable by Shorebird. If you see this
 warning, you should be very careful about publishing your patch, as it may
 cause your app to crash when the Dart code tries to call into native code
 which operates differently than expected. See
-[the next section](#unexpected-native-changes).
+[the next section](#i-see-a-the-release-artifact-contains-native-changes-warning-when-running-shorebird-patch-even-though-i-havent-changed-swiftobjective-ckotlinjava-code-unexpected-native-changes).
```

Comment:
Wrong one?

Reply:
The old one was removed so I think this is correct (or we can remove the link).